plexapi/base.py

```diff
@@ -538,6 +538,7 @@ def _loadData(self, data):
         self.session = self.findItems(data, etag='Session')                         # session
         self.viewedAt = utils.toDatetime(data.attrib.get('viewedAt'))               # history
         self.accountID = utils.cast(int, data.attrib.get('accountID'))              # history
+        self.deviceID = utils.cast(int, data.attrib.get('deviceID'))                # history
```

Comment:
Could you also add `accountID` and `deviceID` to the doc string?

Reply:
f6596a03cea3e69a1e1e93aa29141fe261d25720